### DISCUSSION TITLE
Time series plugin tooltip doesn't have the static class name, id, or attribute

### DISCUSSION THREAD

**Discussion by kamil-homer** [grafana/grafana#41539] [General]
**What happened**:
Currently, the **Time series** tooltip doesn't have any readable (user-friendly) HTML selector, besides generated CSS class ("css-tfew28"). I want to access the tooltip's DOM element, but I can access it only via this CSS class and I assume that the class can change in the future.

**What you expected to happen**:
It will be good if the tooltip would have a selector of any sort - constant class name, id, attribute. Similar to the old Graph, which at least has a 'tooltip' class assigned.

**How to reproduce it**:
The issue can be viewed on the test website, by inspecting the DOM.
https://play.grafana.org/d/000000016/1-time-series-graphs?orgId=1

![image](https://user-images.githubusercontent.com/31961734/141093903-c3cbcb12-c0be-4218-8de8-031a81e87608.png)

**Environment**:
- Grafana version: 8.3.0
- Grafana plugins: Time series

**Comment by torkelo** [comment]
why do you need this?

**Reply by kamil-homer** [reply]
I'm interacting with the tooltip from my own plugin. I also extended the tooltip's functionality by adding a new child node to it.
Code fragment:
``` 
const getTooltipNode = () => document.querySelector('.css-tfew28') as HTMLElement

const getSelectedTimestamp = () => {
    const tooltip = getTooltipNode()
    if (tooltip) {
      setTimestamp((tooltip.firstChild as HTMLElement).innerHTML)
    }
}
```

Works as expected, but may break with the upcoming Grafana updates as the CSS class may change. I think that this small feature can be helpful in the future.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
